keps/sig-node/4639-oci-volume-source/README.md

```diff
@@ -858,6 +849,37 @@ in back-to-back releases.
 - Deprecate the flag
 -->
 
+#### Alpha
+
+- Initial implementation added
+- CRI implementations starting to add support
+
+#### Beta
+
+- Add support for [`subPath`](https://kubernetes.io/docs/concepts/storage/volumes/#using-subpath):
+    - By adding a new CRI string `image_sub_path` field to the
+      [`Mount`](https://github.com/kubernetes/cri-api/blob/010fdf8/pkg/apis/runtime/v1/api.proto#L221)
+      message. Alternatively it would be possible to re-use the existing
+      [`Mount.host_path`] field which is empty right now if an image volume is
+      being used.
+    - If the sub path does not exist in the image, then runtimes are considered
+      to fail and return an error.
+- Expand unit and e2e tests.
+- Add three kubelet metrics for an admin to see the success of their image volumes:
+    - `image_volume_requested_total`
+    - `image_volume_mounted_success`
+    - `image_volume_mounted_error`
+- Production support in at least one of CRI-O and containerd, and a release
+  candidate is available in the other.
+    - However, until both CRI-O and containerd have a released RC with support, this feature will be off by default
```

Comment:
It's odd to specify which specific third party projects must support this in order for an enhancement to graduate.

Reply:
these are new conventions being put together by SIG node to signal feature stability while also giving time for third-party projects to adopt them